processor/servicegraphprocessor/processor.go

```diff
@@ -475,6 +504,17 @@ func (p *serviceGraphProcessor) storeExpirationLoop(d time.Duration) {
 	}
 }
 
+func (p *serviceGraphProcessor) getPeerHost(m []string, peers map[string]string) string {
+	peerStr := "unknown"
+	for _, s := range m {
+		if len(peers[s]) != 0 {
+			peerStr = peers[s]
```

Comment:
Nitpick: I think this is more idiomatic.

```suggestion
		if peer, ok := peers[s]; ok {
			peerStr = peer
```